README.md

```diff
@@ -78,6 +78,15 @@ Right click on your AKS cluster and click on **Configure CI/CD Workflow** to cre
 
 Right click on your AKS cluster and click on **Browse CI/CD Workflows** to view details of all existing Continous Integration / Continous Deployment (CI/CD) pipelines that are configured against your cluster.
 
+### Install Azure Service Operator
+
+Right click on your AKS cluster and click on **Install Azure Service Operator** to easily deploy the latest version of Azure Service Operator (ASO) on your AKS cluster and provision and connect applications to Azure resources within Kubernetes. Upon selecting the option, a web view will load requesting your input on Service Principal App Id and the password. Fill out these required fields and click **Submit** to kick off the installation of Azure Servic Operator.
```

Comment:
💡 Idea share: Shall we move the below part somewhere at the top. (just idea float, main motivation is that user will have moment to know ASO first before they follow instruction. What do you guys think?

> For more information on Azure Service Operator, visit [Azure Service Operator (for Kubernetes)](https://github.com/Azure/azure-service-operator). If you are experiencing issues with Azure Service Operator, visit [Azure Service Operator (ASO) troubleshooting](https://github.com/Azure/azure-service-operator/docs/troubleshooting.md).

Reply:
Hi Tats, thank you for the suggestion! I actually considered it too but kept it like this to keep the format consistent with the other sections. Other sections also start off with instructions and have links to learn more at the end.